src/MoBi.Assets/AppConstants.cs

```diff
@@ -1993,6 +1991,11 @@ public static string TheModuleWillBeConvertedFromPKSimToExtensionModule(string m
          {
             return $"The PK-Sim module '{moduleName}' will be converted to an extension module";
          }
+
+         public static string NewEntitiesBuildingBlock(string typeFor)
+         {
+            throw new NotImplementedException();
```

Comment:
`NotImplementedException` ??

Reply:
Was not used. I added it when I was still in formulation stages lol